.github/workflows/windows1.yml

```diff
@@ -0,0 +1,115 @@
+
+      - name: Set LIB for Intel oneAPI
+        run: echo "LIB=C:\Program Files (x86)\Intel\oneAPI\compiler\2024.1\lib;$env:LIB" | Out-File -FilePath $env:GITHUB_ENV -Append
+
+      - name: Set LIB for MSVC
+        run: echo "LIB=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.40.33807\ATLMFC\lib\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.40.33807\lib\x64;C:\Program Files (x86)\Windows Kits\10\lib\10.0.22621.0\ucrt\x64;C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x64;$env:LIB" | Out-File -FilePath $env:GITHUB_ENV -Append
+
+      - name: Set INCLUDE for MSVC
+        run: echo "INCLUDE=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.40.33807\include;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.40.33807\ATLMFC\include;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\VS\include;C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt;$env:INCLUDE" | Out-File -FilePath $env:GITHUB_ENV -Append
+
+      - name: set FC
+        run: echo "FC=ifx" | Out-File -FilePath $env:GITHUB_ENV -Append
+
+      - name: Build
+        run: python dev.py build -C-Duse-pythran=false -C--vsenv
```

Comment:
why pythran=false? what fails if we use it? (this is fine for this PR, but longterm we wanted to make pythran non-optional)

Reply:
MSVC is unable compile Pythran code. So I and @rgommers decided to set `use-pythran=false`.

Reply:
Did you open the Pythran issue we talked about? If not, it'd be great to do that. And and a code comment above this line linking to it, so it's clear why we're adding `use-pythran=false`